### PR TITLE
Events for TWiR 447

### PR DESCRIPTION
**Pull request by mariannegoldin**

### CODE REVIEW

File: draft/2022-06-15-this-week-in-rust.md [hunk 1]
```diff
@@ -89,30 +89,16 @@ Rusty Events between 2022-06-15 - 2022-07-13 🦀
...
-    * [**Second Tuesday**](https://www.meetup.com/Dallas-Rust/events/vqtjcsydcjbsb/)
-* 2022-06-14 | Rostock, DE | [Altow Academy](https://www.meetup.com/altow-academy/)
-    * [**Rust Meetup Rostock**](https://www.meetup.com/altow-academy/events/286138086/)
-* 2022-06-14 | Saarbrücken, DE | [Rust-Saar](https://www.meetup.com/rust-saar/)
-    * [**Meetup: 22u16**](https://www.meetup.com/rust-saar/events/286291318/)
 * 2022-06-15 | Philadelphia, PA, US | [Rust Philly (Rust Philadelphia)](https://www.meetup.com/RustPhilly/)
     * [**Remote Book Club: Rust for Rustaceans Chapter Discussion**](https://www.meetup.com/RustPhilly/events/qkbktsydcjbtb/)
 * 2022-06-15 | Vancouver, BC, CA | [Vancouver Rust](https://www.meetup.com/Vancouver-Rust/)
     * [**Nushell**](https://www.meetup.com/Vancouver-Rust/events/nwcmpsydcjbtb/)
+* 2022-06-17 [Tehran, IR] | [Rust Iran Meetup](https://rust-meetup.ir/)
```
Comment (andrewpollack) [inline draft/2022-06-15-this-week-in-rust.md:96]:
```suggestion
* 2022-06-17 | Tehran, IR | [Rust Iran Meetup](https://rust-meetup.ir/)
```

File: draft/2022-06-15-this-week-in-rust.md [hunk 2]
```diff
@@ -129,37 +115,39 @@ Rusty Events between 2022-06-15 - 2022-07-13 🦀
     * [**Buffalo Rust User Group, First Tuesdays**](https://www.meetup.com/buffalo-rust-meetup/events/xgmfssydckbhb/)
 * 2022-07-06 | Indianapolis, IN, US | [Indy Rust](https://www.meetup.com/indyrs/)
     * [**Indy.rs - with Social Distancing**](https://www.meetup.com/indyrs/events/qwtdjsydckbjb/)
+* 2022-07-07 | Charlottesville, VA, US | [Charlottesville Rust Meetup](https://www.meetup.com/charlottesville-rust-meetup/)
+    * [**Rust, nalgebra, and Fourier Optics**](https://www.meetup.com/charlottesville-rust-meetup/events/285818136/)
+* 2022-07-09 | Online | [Rust Game Dev](https://github.com/rust-gamedev/wg)
```
Comment (andrewpollack) [inline draft/2022-06-15-this-week-in-rust.md:120]:
```suggestion
* 2022-07-09 | Virtual | [Rust Game Dev](https://github.com/rust-gamedev/wg)
```
Suggesting using term "Virtual" to stay consistent with title. Open to either!

**Review by andrewpollack — CHANGES_REQUESTED** [review]

File: draft/2022-06-15-this-week-in-rust.md (hunk 1 above)
Comment (mariannegoldin) [reply draft/2022-06-15-this-week-in-rust.md:96]: Thanks for the catch!

File: draft/2022-06-15-this-week-in-rust.md [hunk 3]
```diff
@@ -89,30 +89,16 @@ Rusty Events between 2022-06-15 - 2022-07-13 🦀
...
 * 2022-06-15 | Philadelphia, PA, US | [Rust Philly (Rust Philadelphia)](https://www.meetup.com/RustPhilly/)
     * [**Remote Book Club: Rust for Rustaceans Chapter Discussion**](https://www.meetup.com/RustPhilly/events/qkbktsydcjbtb/)
 * 2022-06-15 | Vancouver, BC, CA | [Vancouver Rust](https://www.meetup.com/Vancouver-Rust/)
     * [**Nushell**](https://www.meetup.com/Vancouver-Rust/events/nwcmpsydcjbtb/)
+* 2022-06-17 | Tehran, IR | [Rust Iran Meetup](https://rust-meetup.ir/)
+    * [**Rust Iran Meetup #7 - Actix-Web Framework**](https://rust-meetup.ir/2022/06/17/seventh-meetup.html)
+* 2022-06-20 | Köln, DE | [Rust Cologne](https://www.meetup.com/rustcologne/)
+    * [**Reboot**](https://www.meetup.com/rustcologne/events/286557151/)
+* 2022-06-21 | Berlin, DE | [Berline.rs](https://berline.rs/)
+    * [** Rust Hack and Learn**](https://berline.rs/2022/06/21/rust-hack-and-learn.html)
```
Comment (mariannegoldin) [inline draft/2022-06-15-this-week-in-rust.md:101]:
```suggestion
    * [**Rust Hack and Learn**](https://berline.rs/2022/06/21/rust-hack-and-learn.html)
```

**Pull request merged**